server/controllers/finance/account.js

```diff
@@ -0,0 +1,80 @@
+var db = require('../../lib/db');
+
+function create (req, res, next){
+  var record = req.body;
+  var create_account_query = 'INSERT INTO account SET ?';
+  var transaction = db.transaction();
+
+  transaction
+  .addQuery(create_account_query, [record]);
+
+  transaction.execute()
+  .then(function (results){
+    var confirmation = results;
+    res.status(201).json(confirmation);
+    return;
+  })
+  .catch(next)
+  .done();
+}
+
+function update (req, res, next){
+  var queryData = req.body;
+  var accountId = req.params.id;
+  var update_account_query = 'UPDATE account SET ? WHERE id = ?';
+
+  db.exec(update_account_query, [queryData, accountId])
```

Comment:
What happens if I send a PUT request with a new id?  For example "PUT /accounts/203  { id : 2000 }".  I think this route will allow you to change the database ID of any account...  Is this a good idea?  Should the API prevent this?